src/main/java/org/opensearch/securityanalytics/threatIntel/resthandler/RestGetIocFindingsAction.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.securityanalytics.threatIntel.resthandler;
+
+import org.opensearch.client.node.NodeClient;
+import org.opensearch.commons.alerting.model.Table;
+import org.opensearch.rest.BaseRestHandler;
+import org.opensearch.rest.RestRequest;
+import org.opensearch.rest.action.RestToXContentListener;
+import org.opensearch.securityanalytics.SecurityAnalyticsPlugin;
+import org.opensearch.securityanalytics.action.GetFindingsAction;
+import org.opensearch.securityanalytics.threatIntel.action.GetIocFindingsRequest;
+
+import java.io.IOException;
+import java.time.DateTimeException;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.List;
+
+import static java.util.Collections.singletonList;
+import static org.opensearch.rest.RestRequest.Method.GET;
+
+public class RestGetIocFindingsAction extends BaseRestHandler {
+
+    @Override
+    public String getName() {
+        return "get_ioc_findings_action_sa";
+    }
+
+    @Override
+    protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient client) throws IOException {
+        String threatIntelMonitorId = request.param("monitor_id");
+
+        String sortString = request.param("sortString", "timestamp");
+        String sortOrder = request.param("sortOrder", "asc");
+        String missing = request.param("missing");
```

Comment:
what is this?

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/current/sort-search-results.html#_missing_values